### ISSUE TITLE
`tmux` script keeps creating new sessions because it can't attach to the session specified

### ISSUE THREAD

**Issue by jetejetteraiASAP** [jetejetteraiASAP/setups#4]
Best show with an example.

```
~ $ tmux ls
config-1: 1 windows (group config)
work-0: 1 windows (group work)
~ $ cont
can't find session: config
~ $ tmux ls
config-1: 1 windows (group config)
config-3: 1 windows (group config)
work-0: 1 windows (group work)
work-2: 1 windows (group work)
```
I always have to kill the server when it reaches this stage. It is not able to attach to the right session when it's already running, and creates a new session again.

*I deleted the date and time which `tmux` shows for each session, rest assured that I can see those.*

**Comment by jetejetteraiASAP** [comment]
All work on this issue is shifted to the [tmux-dev](https://github.com/jetejetteraiASAP/setups/blob/tmux-dev/bash/.bash/.bash_functions.sh) branch.

Need to script `tmux` according to the arguments passed. Need to get the output of `tmux ls | grep -Po "${1}" | head -1` and compare with input when I'm trying to create a new session, and attach only if session does not exist.

**Comment by jetejetteraiASAP** [comment]
Fixed in commit 0c5ce50facef71bca7f26ae99a99cd982787763d. `tmux` now auto attaches, or creates 2 different sessions (which are hardcoded, I don't see a need to dynamically assign them from inputs).
